setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-Dummy-Country',
-    version = '0.1.2',
+    version = '1.0.0',
```

Comment:
We don't apply semver for the dummy country package, so we should stay at sub-1.0 level versions.